certified-connectors/IA-Connect Session/apiDefinition.swagger.json

```diff
@@ -4173,6 +4185,658 @@
+                  "type": "string",
+                  "x-ms-visibility": "advanced",
+                  "default": "@{workflow()}",
+                  "title": "Workflow",
+                  "description": "Add the following expression here: workflow()"
+                }
+              },
+              "required": [
+                "RemoteAddress",
+                "OutputFolderPath",
+                "RDPFileName",
+                "Workflow"
+              ]
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "RDPFilePath": {
+                  "type": "string",
+                  "title": "RDP file path",
+                  "description": "The full path of the generated RDP file."
+                }
+              }
+            }
+          }
+        },
+        "operationId": "GenerateRDPFile"
+      }
+    },
+    "/Environment/LaunchRemoteDesktopSession": {
+      "post": {
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "summary": "Launch Remote Desktop Session (RDS)",
+        "description": "Launch a Remote Desktop Session (RDS) using a RDP launch file.",
+        "parameters": [
+          {
+            "in": "body",
+            "name": "LaunchRemoteDesktopSession",
+            "description": "Inputs for IA-Connect LaunchRemoteDesktopSession action",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "RDPFilePath": {
+                  "type": "string",
+                  "title": "RDP file path",
+                  "description": "The full path to a RDP file (which could either pre-exist or be created using the action: Create Remote Desktop Session launch file)."
+                },
+                "TrustRemoteComputer": {
+                  "type": "boolean",
+                  "default": true,
+                  "title": "Trust remote computer",
+                  "description": "Set to true to mark the remote address (set in the RDP file) as trusted (in the Windows registry), mainly for device redirection purposes, eliminating a popup you might receive when launching the RDP file."
+                },
+                "Workflow": {
+                  "type": "string",
+                  "x-ms-visibility": "advanced",
+                  "default": "@{workflow()}",
+                  "title": "Workflow",
+                  "description": "Add the following expression here: workflow()"
+                }
+              },
+              "required": [
+                "RDPFilePath",
+                "Workflow"
+              ]
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {}

```

Comment:
@LukeMcGovern-UL Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

Reply:
@sriyen-msft Thank you for reviewing our pull request - in this scenario the command returns no output and therefore has no output schema. We distinguish between success or failure by returning the relevant HTML code, 200 (success) or 400 (failure) which will result in an exception in the Power Automate Flow. 

Most of our actions do return data and all of these have an appropriate output schema defined.